contracts/src/v0.8/functions/dev/1_0_0/FunctionsBilling.sol

```diff
@@ -167,7 +167,7 @@ abstract contract FunctionsBilling is Routable, IFunctionsBilling {
 
     uint256 juelsPerGas = _getJuelsPerGas(gasPriceWithOverestimation);
     uint256 estimatedGasReimbursement = juelsPerGas * executionGas;
-    uint96 fees = donFee + adminFee;
+    uint96 fees = uint96(donFee + adminFee);
```

Comment:
I believe you have to cast each one separately, before adding.

Reply:
Yeah, you're right, with that change my little unit test doesn't revert
```
function testUint72Success() public {
    uint72 maxVal = 4_722_366_482_869_645_213_695;
    uint96 bad = uint96(maxVal) + uint96(maxVal);
} 
```